### PR TITLE
feat: Add new web languages

### PR DESCRIPTION
**Pull request by FnrDev**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,7 +58,21 @@ module.exports.webLangs = {
   ro: { name: "Română", flag: "ro" },
   sk: { name: "Slovák", flag: "sk" },
   ja: { name: "日本語", flag: "jp" },
-  hu: { name: "Magyar", flag: "hu" }
+  hu: { name: "Magyar", flag: "hu" },
+  ko: { name: "한국어", flag: "ko" },
+  lt: { name: "Italian", flag: "lt" },
+  no: { name: "Norsk", flag: "no" },
+  pl: { name: "Polish", flag: "pl" },
```
Comment (Dramex) [inline index.js:65]: Polski

File: index.js [hunk 2]
```diff
@@ -58,7 +58,21 @@ module.exports.webLangs = {
...
+  pt_BR: { name: "português", flag: "pt_BR" },
```
Comment (Dramex) [inline index.js:66]: portuguesa, brasileira

File: index.js (hunk 1 above)
Comment (FnrDev) [reply index.js:65]: done

File: index.js (hunk 2 above)
Comment (FnrDev) [reply index.js]: fixed

**Pull request merged**